test/fixture.html

```diff
@@ -1 +1,15 @@
-<h1>unicorns</h1>
+<html>
+<head>
+	<style>
+	#team { padding: 0px; height: 80px; width: 1024px; }
```

Comment:
If this CSS line is not provided and we capture the dom `#team` element, the width and height of the output file are the size of that element, not the size we provided in the `src` function. Even when the `width` is set to `1024px`, but the padding is not set to `0px`, the output size is something like `1080px` wide. Might be related to #141.